### PR TITLE
:memo: updated README

### PR DESCRIPTION
**Pull request by Lunarmagpie**
### Changes
1. Fixed the first example to actually run
2. Remove the security warning in logging because we never had one afaik.

 ### Check off the following

-   [x] I have tested my changes with the current requirements
-   [x] My Code follows the pep8 code style.

### CODE REVIEW

File: docs/PYPI.md [hunk 1]
```diff
@@ -1,190 +0,0 @@
...
-[in our docs](https://pincer.readthedocs.io/en/latest/pincer.html#pincer.client.middleware)
-._
-
-## 🏷️ License
-
-`© 2021 copyright Pincer`
-
-This repository is licensed under the MIT License.
-
-See LICENSE for details.
```
Comment (Sigmanificient) [inline docs/PYPI.md:190]: I'd prefer to stay with a separated README for PyPI as he dont support table display or banner from the repo README

**Review by Sigmanificient — CHANGES_REQUESTED** [review]

File: docs/PYPI.md (hunk 1 above)
Comment (Lunarmagpie) [reply docs/PYPI.md]: I reverted the change.

**Comment by Lunarmagpie** [comment]
I think maintaining 2 READMEs is a terrible idea. Its be pretty easy to fix by using setuptools though but im not sure if we really want to add that complexity. Just putting the idea out there.

File: docs/PYPI.md [hunk 2]
```diff
@@ -74,61 +82,44 @@ following:
...
+client = Bot("YOUR_TOKEN_HERE")
 client.run()
 ```
 
-### Inherited client
+### Interactions
 
-You have the possibility to use your own class to inherit from the Pincer bot
-base.
+Pincer is designed to make developing application commands intuitive and fast.
```
Comment (trag1c) [inline docs/PYPI.md:114]: Not a big fan of this *"Pincer is designed to [...]"* repetition, would refactor the sentence. Also both sentences don't sound the best in my opinion.

File: docs/PYPI.md [hunk 3]
```diff
@@ -137,20 +128,33 @@ class Bot(Client):
     async def say(self, message: str):
         return message
 
+    @user_command
+    async def user_command(self, user: User):
+        return f"The user is {user}"
+
+    @message_command(name="Message command")
+    async def message_command(self, message: UserMessage):
+        return f"The message read \"{message.content}\""
```
Comment (trag1c) [inline docs/PYPI.md:137]:
```suggestion
        return f'The message read "{message.content}"'
```

File: docs/PYPI.md [hunk 4]
```diff
@@ -137,20 +128,33 @@ class Bot(Client):
...
+
     @command(description="Add two numbers!")
     async def add(
             self,
             first: CommandArg[int, Description["The first number"]],
             second: CommandArg[int, Description["The second number"]]
     ):
```
Comment (trag1c) [inline docs/PYPI.md:144]:
```suggestion
    async def add(
        self,
        first: CommandArg[int, Description["The first number"]],
        second: CommandArg[int, Description["The second number"]]
    ):
```

File: docs/PYPI.md [hunk 5]
```diff
@@ -137,20 +128,33 @@ class Bot(Client):
...
         return f"The addition of `{first}` and `{second}` is `{first + second}`"
+
+
 ```
 
 For more examples you can take a look at the examples folder or check out our
-bot on GitHub:
+bot:
```
Comment (trag1c) [inline docs/PYPI.md:151]: Would move this one word to the previous line tbh

File: docs/PYPI.md [hunk 6]
```diff
@@ -137,20 +128,33 @@ class Bot(Client):
...
             self,
             first: CommandArg[int, Description["The first number"]],
             second: CommandArg[int, Description["The second number"]]
     ):
         return f"The addition of `{first}` and `{second}` is `{first + second}`"
+
+
 ```
 
 For more examples you can take a look at the examples folder or check out our
```
Comment (trag1c) [inline docs/PYPI.md:150]:
```suggestion
For more examples, you can take a look at the examples folder or check out our
```

File: docs/PYPI.md [hunk 7]
```diff
@@ -11,30 +11,38 @@
...
 ![GitHub](https://img.shields.io/github/license/Pincer-org/Pincer)
 ![Discord](https://img.shields.io/discord/881531065859190804)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-An asynchronous Python API wrapper meant to replace discord.py
+# <img src="../assets/svg/pincer.svg" height="24px" alt="Pincer Logo"> Pincer
+The snappy asynchronous discord api wrapper API wrapper written with aiohttp & websockets.
 
-## The package is currently within the pre-alpha phase
+| :exclamation: | The package is currently within alpha phase |
```
Comment (trag1c) [inline docs/PYPI.md:22]:
```suggestion
| :exclamation: | The package is currently within the Alpha phase |
```

File: docs/PYPI.md [hunk 8]
```diff
@@ -11,30 +11,38 @@
...
 ## ☄️ Installation
 
 Use the following command to install Pincer into your Python environment:
 
-```bash
+```sh
 pip install pincer
 ```
 
+To install our version with Aiohttp Speedup do:
```
Comment (trag1c) [inline docs/PYPI.md:40]:
```suggestion
To install our version with Aiohttp Speedup, use:
```

File: docs/PYPI.md [hunk 9]
```diff
@@ -11,30 +11,38 @@
...
 ![GitHub last commit](https://img.shields.io/github/last-commit/Pincer-org/Pincer)
-![GitHub commit activity](https://img.shields.io/github/commit-activity/m/Pincer-org/Pincer)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/m/Pincer-org/Pincer?label=commits)
 ![GitHub](https://img.shields.io/github/license/Pincer-org/Pincer)
 ![Discord](https://img.shields.io/discord/881531065859190804)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-An asynchronous Python API wrapper meant to replace discord.py
+# <img src="../assets/svg/pincer.svg" height="24px" alt="Pincer Logo"> Pincer
+The snappy asynchronous discord api wrapper API wrapper written with aiohttp & websockets.
```
Comment (trag1c) [inline docs/PYPI.md:20]: Would already remove the `& websockets` part so there's no need to update it later when #329 gets merged, but that's up to you I guess, so use one of these:
```suggestion
The snappy asynchronous Discord API wrapper written with aiohttp & websockets.
```
```suggestion
The snappy asynchronous Discord API wrapper written with aiohttp.
```

File: docs/PYPI.md [hunk 10]
```diff
@@ -165,9 +169,6 @@ import logging
 logging.basicConfig(level=logging.DEBUG)
 ```
 
-**Note:** _A lot of printing can happen, including sensitive information, so
-make sure to be aware of what you're doing if you're enabling it!_
-
 #### Middleware
 
 _From version 0.4.0-dev, the middleware system has been introduced. This system
```
Comment (trag1c) [inline docs/PYPI.md:174]: For some reason I can't select all lines here so I'll just paste my suggestion
```markdown
The middleware system was introduced in version `0.4.0-dev`. This system gives you the
freedom to create custom events and remove the already existing middleware created by
the developers. Your custom middleware directly receives the payload from
Discord. You can't do anything wrong without accessing the `override` attribute, but if
you do access it, the Pincer team will not provide any support for weird behavior.
So, in short, only use this if you know what you're doing. An example of using
the middleware system (with a custom `on_ready` event) can be found
[in our docs](https://pincer.readthedocs.io/en/latest/pincer.html#pincer.client.middleware).
```

**Review by trag1c — CHANGES_REQUESTED** [review]
All suggestions apply to `README.md` as well.

File: docs/PYPI.md (hunk 9 above)
Comment (Lunarmagpie) [reply docs/PYPI.md:20]: Good idea, I hate merge conflicts xD

File: docs/PYPI.md (hunk 10 above)
Comment (Lunarmagpie) [reply docs/PYPI.md:174]: I like this except for `on_ready` being in parenthesis so I won't keep that

Comment (trag1c) [reply docs/PYPI.md:174]: Sure

File: docs/PYPI.md (hunk 3 above)
Comment (Lunarmagpie) [reply docs/PYPI.md:137]: I don't really like using '...' for quotes :skull: I'd rather flip them if we need to get rid of the escape character.

File: docs/PYPI.md (hunk 2 above)
Comment (Lunarmagpie) [reply docs/PYPI.md:114]: > Pincer makes developing application commands intuitive and fast.
:thought_balloon:

File: docs/PYPI.md (hunk 3 above)
Comment (trag1c) [reply docs/PYPI.md:137]: Yeah, flip them then, not a fan of using single quotes either. I'm not sure but I think PEP8 prefers `'very "funny"'` over `"very \"funny\""`, at least that's what autopep8 for me (unless I totally messed up and it was a different formatter).

File: docs/PYPI.md (hunk 2 above)
Comment (Lunarmagpie) [reply docs/PYPI.md:111]: Im gonna go with that

File: docs/PYPI.md [hunk 11]
```diff
@@ -74,61 +79,44 @@ following:
...
 
 # Note that both `Bot` and `Client` are valid!
-bot = Bot("...")
+bot = Bot("YOUR_TOKEN_HERE")
 bot.run()
 ```
 
 **An example on the `on_ready` event**
 
+Pincer makes developing application commands intuitive and fast.
```
Comment (trag1c) [inline docs/PYPI.md:94]: <details>
<summary>?</summary>
duplicate of L111
</details>
<i>(applies to both files)</i>

**Review by trag1c — APPROVED** [review]

File: docs/PYPI.md [hunk 12]
```diff
@@ -11,30 +11,35 @@
...
+> <img src="../assets/svg/discord.svg" width="16px" alt="Discord Logo"> ｜Join the Discord server: https://discord.gg/pincer <br>
+> <img src="../assets/svg/pypi.svg" width="16px" alt="PyPI Logo"> ｜The PyPI package: https://pypi.org/project/Pincer <br>
+> <img src="../assets/svg/pincer.svg" width="16px" alt="Pincer Logo"> ｜Our website: https://pincer.dev <br>
+> 📝 | ReadTheDocs: https://pincer.readthedocs.io
 
 ## ☄️ Installation
 
 Use the following command to install Pincer into your Python environment:
 
-```bash
```
Comment (Sigmanificient) [inline docs/PYPI.md:34]: omg wtf is this xD
Thx you so much

File: docs/PYPI.md [hunk 13]
```diff
@@ -11,30 +11,35 @@
...
+```sh
 pip install pincer
 ```
 
+To install our version with Aiohttp Speedup, use:
+
+```sh
+pip install pincer[speed]
+```
+
```
Comment (Sigmanificient) [inline docs/PYPI.md:42]: There is also `img` but instead of saying "to install ..." for all of them maybe should we put a table or something?

File: docs/README.md [hunk 14]
```diff
@@ -82,93 +82,79 @@ following:
...
-
-@client.event
-async def on_ready():
-    print(f"Logged in as {client.bot} after {perf_counter()} seconds")
+from pincer import Client
 
+class Bot(Client):
+    @Client.event
+    async def on_ready():
+        print(f"Logged in as {client.bot} after {perf_counter()} seconds")
```
Comment (Sigmanificient) [inline docs/README.md:106]: shouldn't we use `client` repr? ...client as no repr implemented yet xD

Comment (Sigmanificient) [inline docs/README.md:106]: also `perf_counter()` is broken on my server idk, it put like 3 milion seconds, work fine when subtracting from a marker

**Review by Sigmanificient — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 14 above)
Comment (Lunarmagpie) [reply docs/README.md:106]: Im not doing this this PR cause I actually want it to be merged :skull: 
and i think its good how it is

Comment (Lunarmagpie) [reply docs/README.md:106]: It works fine locally? Idk what Im supposed to do lol

File: docs/PYPI.md (hunk 13 above)
Comment (Lunarmagpie) [reply docs/PYPI.md:39]: This is outside the scope of what I want to do in this PR

File: docs/PYPI.md (hunk 11 above)
Comment (Lunarmagpie) [reply docs/PYPI.md]: I changed the first instance to "Pincer bots are required to inherit from the Client."

File: docs/PYPI.md [hunk 15]
```diff
@@ -91,7 +88,7 @@ bot.run()
 
 **An example on the `on_ready` event**
 
-Pincer makes developing application commands intuitive and fast.
+Pincer bots are required to inherit from the Client.
```
Comment (Sigmanificient) [inline docs/PYPI.md:91]: ye lol md template woudln't be too bad xD

File: docs/README.md (hunk 14 above)
Comment (Lunarmagpie) [reply docs/README.md:106]: nvm its supposed to be ms not seconds

**Comment by Lunarmagpie** [comment]
I'm merging. The stuff that I didn't change but are requested changes I think should be done in a different PR since I don't want to be stuck maintaining this branch for a while.
